app/src/NativeBase/Components/Forms/DatePicker.tsx

```diff
@@ -109,7 +109,7 @@ const DatePicker: FC<DatePickerProps> = ({
           <Text
             paddingTop="8"
             fontSize="sm"
-            fontFamily="primarySemiBold"
+            fontWeight="600"
```

Comment:
With all these changes from `fontFamily` to a `fontWeight` setting, Pam only just changed these -- assume you've checked with her or have a good reason to change them again?

Reply:
Your changes here are reversing some changes Pam made to fonts -- e.g. here you can see how it's set on the `develop` branch: https://github.com/Scottish-Tech-Army/Volunteer-app/blob/develop/app/src/NativeBase/Components/Forms/DatePicker.tsx#L112